python/src/IndependentMetropolisHastings_doc.i.in

```diff
@@ -0,0 +1,77 @@
+%feature("docstring") OT::IndependentMetropolisHastings
+"Random Walk Metropolis-Hastings method.
```

Comment:
that's going to be confusing :)

Reply:
For sure! We have replaced Random-Walk by Independent...